content/mysql/concepts/query-attributes/query-attributes.md

```diff
@@ -0,0 +1,48 @@
+---
+Title: 'Query-Attributes'
+Description: 'Metadata associated with SQL queries in MySQL for plugins, debugging, or tracing without altering query behavior.'
+Subjects:
+  - 'Attributes'
+  - 'Metadata'
+  - 'SQL'
+  - 'MySQL'
+Tags:
+  - 'MySQL'
+  - 'Database'
+  - 'Tables'
+  - 'Data'
+CatalogContent:
+  - 'learn-sql'
+  - 'paths/analyze-data-with-sql'
+---
+
+**Query attributes**, in MySQL allows you to add metadata to SQL queries. They help in passing custom information, debugging, and tracing query execution. These attributes do not change the behavior of the query but provide additional context for tools or services interacting with the database.
+
+## Syntax
+
+In MySQL, query attributes are typically set using the `SET` statement. The syntax for adding a query attribute is as follows:
+
+```pseudo
+SET [GLOBAL|SESSION] query_attribute_name = 'value';
+```
+- `GLOBAL` or `SESSION`: Specifies whether the attribute applies globally for all sessions or just for the current session.
+- `query_attribute_name`: Name of the query attribute.
+- `value`: Represents the value assigned to the attribute. 
+ 
+## Example
+
+In the following example we have have set a custom query attribute that sets a rule that says the query should not take longer than `10` seconds to run. If it does, MySQL will stop it
+
+```sql
+-- Set a query attribute 
+SET SESSION query_time_limit = '10s';
+-- Using the attribute in query
+SELECT * FROM orders WHERE order_date > '2024-01-01';
```

Comment:
The example code doesn't correctly demonstrate the session time limit functionality, the `SET SESSION` part will have no effect here.